smart_news_query_embeddings/preprocessing/bert_tokenizer.py

```diff
@@ -96,27 +101,37 @@ def tokenize_data(inputs, labels, tokenizer, max_seq_length, num_classes):
         train_labels.append(one_hot)
 
     train_tokens = map(tokenizer.tokenize, inputs)
-    train_tokens = map(lambda tok: ["[CLS]"] + tok + ["[SEP]"], train_tokens)
+    train_tokens = map(lambda tok: [CLS] + tok + [SEP], train_tokens)
     train_token_ids = list(map(tokenizer.convert_tokens_to_ids, train_tokens))
 
     train_token_ids = map(
-        lambda tids: tids + [0] * (max_seq_length - len(tids)) if len(tids) <= 128 else tids[:128],
+        lambda tids: tids + [0] * (max_seq_length - len(tids))
+        if len(tids) <= max_seq_length else tids[:max_seq_length],
         train_token_ids)
     train_token_ids = list(train_token_ids)
-    print(type(train_token_ids), type(train_token_ids[0]))
     train_token_ids = np.array(train_token_ids)
 
     train_labels_final = np.array(train_labels)
 
     return train_token_ids, train_labels_final
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--data-path', '-d', defualt='nyt_data_from_2015.pkl', type=str)
+
+    args = parser.parse_args()
+
     tokenizer = create_tokenizer('uncased_L-12_H-768_A-12')
 
-    df = get_filtered_nyt_data('nyt_data_from_2015.pkl')
+    df = get_filtered_nyt_data(args.data_path)
     df['category_labels'] = df['section'].astype('category').cat.codes
```

Comment:
Add a comment what the column data was how it was processed and how it will be used in training.